spec/factories/related_instructions.rb

```diff
@@ -5,5 +5,6 @@
     elective { false }
     sort_order { 1 }
     occupation_standard
+    association :default_course, factory: :course
```

Comment:
Let's leave this out of the factory since it's not required. The specs run faster when fewer database records are added, and we don't want to unnecessarily be adding a course record every time we create a RelatedInstruction object.